Pi2BoardPkg/AcpiTables/Common/Tpm2DeviceSpi0.asl

```diff
@@ -0,0 +1,46 @@
+//
+//  Copyright (c) 2019, Microsoft Corporation. All rights reserved.
+//
+//  This program and the accompanying materials
+//  are licensed and made available under the terms and conditions of the BSD License
+//  which accompanies this distribution.  The full text of the license may be found at
+//  http://opensource.org/licenses/bsd-license.php
+//
+//  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+//
+
+//
+// SSDT definition for a discrete TPM2 device on SPI0
+//
+
+DefinitionBlock ("TPM2Dev.dat", "SSDT", 1, "MSFT", "SPITPM", 1)
+{
+    Scope (\_SB)
+    {
+        Device(TPM1)
+        {
+            Name(_HID, "MSFT0101")
+            Name(_UID, 1)
+            Name(_DDN, "SPI TPM 2.0 Device")            // _DDN: DOS Device Name
+            Name(_STR, Unicode ("SPI TPM 2.0 Device"))  // _STR: Description String
+            Name (RBUF, ResourceTemplate ()
+            {
+                // RaspberryPi2 4MHz, SPI0/CS0

```

Comment:
nit: This file is for the Pi3 as well right?

Reply:
Should be applicable on all RPIs so long as the header is the same layout.